docs/Wallet-System.md

```diff
@@ -16,7 +14,14 @@ theoretically be able to scale to hundreds of thousands of
 wallets/accounts/addresses.
 
 Each account can be of a different type. You could have a pubkeyhash account,
-as well as a multisig account, a witness pubkeyhash account, etc.
+as well as a multisig account, a witness pubkeyhash account, etc. This is where
+the bcoin wallet begins to deviate from strict BIP44: Accounts in a bcoin wallet
+(for example) can be arbitrarily configured for multisig and import xpubs from
+cosigners. Accounts can be configured for Segregrated Witness or not, all within
+the same wallet. Bcoin also deviates from strict BIP44 by using a third branch
+for each acoount. Branch `0` and `1` are for `receive` and `change` addresses
+respectively (which is BIP44 standard) but branch `2` is used by bcoin to derive
+[nested SegWit addresses.](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#P2WPKH_nested_in_BIP16_P2SH)
```

Comment:
A backup of the wallet database is necessary to recovery of coins because of several of these, and should likely be mentioned. There are several [documented issues](https://en.wikipedia.org/wiki/LevelDB#Bugs_and_reliability) with LevelDB corruption, and have experienced this myself: https://github.com/bcoin-org/bcoin/issues/682